cinder/volume/drivers/lightos.py

```diff
@@ -1137,10 +1156,34 @@ def __remove_volume_acl(
         if not acl:
             acl.append('ALLOW_NONE')
 
+        ip_acl = data.get('IPAcl')
+        if not ip_acl:
+            LOG.warning('Got LightOS volume without IP ACL?! data: %s', data)
+            return False
+
+        ip_acl = ip_acl.get('values')
+        if not ip_acl:
+            LOG.warning(
+                'Got LightOS volume without IP ACL values?! data: %s', data)
+            return False
+      
+        for ip in host_ips:
+            try:
+                ip_acl.remove(ip)
+            except ValueError:
+                LOG.warning(
+                'Could not find matching ip %s in ip-acl %s of LightOS volume',
+                ip,
+                lightos_volname
```

Comment:
the log message does not match the parameters (should be ip, ip_acl, lightos_volname) with the message format fixed appropriately)